components/connector/docs/api/connectorapi.yaml

```diff
@@ -0,0 +1,70 @@
+openapi: '3.0.0'
+info:
+  version: '1.0.0'
+  title: 'Kyma Connector Service API'
+paths:
+  /health:
+    get:
+      summary: 'Returns health status.'
+      responses:
+        '200':
+          description: 'The service is up.'
+
+  /v1/tokens/resolve:
```

Comment:
`/v1/tokens/resolve` and `/v1/certificate/data/resolve` require headers. You will find all files [here](https://github.com/kyma-incubator/compass/tree/master/components/connector/pkg/oathkeeper)

Reply:
These headers are not really required but we can add them as optional.